cmd/skopeo/login.go

```diff
@@ -21,7 +21,7 @@ func loginCmd(global *globalOptions) *cobra.Command {
 		global: global,
 	}
 	cmd := &cobra.Command{
-		Use:     "login",
+		Use:     "login [command options] REGISTRY",
```

Comment:
The skopeo convention is to emit `[command options]` instead of just `[options]`